src/overlays/actors/ovl_En_Aob_01/z_en_aob_01.c

```diff
@@ -91,105 +109,137 @@ static EnAobStruct D_809C384C[] = {
     { { -4827.0f, 168.0f, 1328.0f }, 115, 4 }, { { -4130.0f, 150.0f, 1367.0f }, 112, 0 },
 };
 
-void func_809C10B0(EnAob01* this, s32 arg1) {
+void EnAob01_Blink(EnAob01* this, s32 indexToRestartBlink) {
```

Comment:
Perhaps `indexToRestartBlnk` -> `maxEyeIndex`? Up to you though

Reply:
Very good point. I didn't know what to name this, but now that we made `EN_AOB01_EYE_MAX`, it's pretty obvious.